python/tests/test_docs.py

```diff
@@ -0,0 +1,121 @@
+from konduit import *
+
+
+def markdown_blocks(file_path, language='python', safe=True):
+    code_blocks = []
+    code_regex = r'^```.*'
+    code_block_open_re = r'^```(`*)({0}){1}$'.format(language, '' if safe else '?')
```

Comment:
Kinda wondering if there's a more robust markdown parser we could use in place of this? Do you see any edge cases coming from this?

Reply:
good point, let me investigate. edge cases can only arrive from human error, we're literally just checking what comes between opening and closing triple backticks (plus a safeguard for the language)